OpenKeychain/src/main/java/org/sufficientlysecure/keychain/receiver/NetworkReceiver.java

```diff
@@ -0,0 +1,54 @@
+package org.sufficientlysecure.keychain.receiver;
+
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.Context;
+import org.sufficientlysecure.keychain.service.KeyserverSyncAdapterService;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import org.sufficientlysecure.keychain.Constants;
+import org.sufficientlysecure.keychain.util.Log;
+
+public class NetworkReceiver extends BroadcastReceiver {
+    private static final String ACTION_UPDATE_ALL = "update_all";
+
+    Context context;
+    public NetworkReceiver(){}
```

Comment:
Sure. I'll incorporate all the changes and do a final test. Thanks.
On 09-Mar-2016 21:17, "Adithya Abraham Philip" <notifications@github.com>
wrote:

> In
> OpenKeychain/src/main/java/org/sufficientlysecure/keychain/receiver/NetworkReceiver.java
> <https://github.com/open-keychain/open-keychain/pull/1767#discussion_r55537972>
> :
>
> > +import android.content.ComponentName;
> > +import android.content.Context;
> > +import org.sufficientlysecure.keychain.service.KeyserverSyncAdapterService;
> > +import android.content.Intent;
> > +import android.content.pm.PackageManager;
> > +import android.net.ConnectivityManager;
> > +import android.net.NetworkInfo;
> > +
> > +import org.sufficientlysecure.keychain.Constants;
> > +import org.sufficientlysecure.keychain.util.Log;
> > +
> > +public class NetworkReceiver extends BroadcastReceiver {
> > +    private static final String ACTION_UPDATE_ALL = "update_all";
> > +
> > +    Context context;
> > +    public NetworkReceiver(){}
>
> AFAIK, this shouldn't be necessary now since there are no other
> constructors, could you remove this and test again?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-keychain/open-keychain/pull/1767/files#r55537972>
> .
>